shared/src/constants/constants.ts

```diff
@@ -410,15 +425,10 @@ export const VaultAddressMap: {
         chainId: CHAINID.ETH_MAINNET,
       },
   // FIXME: change with real addresses
-  "rSOL-THETA": isDevelopment()
-    ? {
-        v2: v2deployment.kovan.RibbonTreasuryVaultPERP,
-        chainId: CHAINID.ETH_KOVAN,
-      }
-    : {
-        v2: v2deployment.mainnet.RibbonTreasuryVaultPERP,
-        chainId: CHAINID.ETH_MAINNET,
-      },
+  "rSOL-THETA": {
+    v2: getSolanaAddresses().vaults["rSOL-THETA"],
+    chainId: 99999999, // we stub this out with a fake chainid
```

Comment:
What is the complication that you are facing that requires a `chainId`? 

To provide context, a lot of the refactors were done so as not to reference `chainId`, but rather the `Chains` enum to move away from the EVM-oriented approach on the frontend

Reply:
I was mostly trying to fit in `VaultAddressMap` without making many changes, since it touches a lot of the codebase